### PR TITLE
implement io.WriterTo on memFile

### PR DESCRIPTION
**Pull request by chonthu**

### CODE REVIEW

**Comment by chonthu** [comment]
Sorry, just noticed your io helper functions for ioutil, closing.

**Comment by chonthu** [comment]
Actually no I cant seem to do a io.Copy on a memfile without this.

File: mem/file.go [hunk 1]
```diff
@@ -29,6 +29,8 @@ import "time"
 const FilePathSeparator = string(filepath.Separator)
 
 type File struct {
+	io.Writer
```
Comment (davecheney) [inline mem/file.go]: This adds a public field, File.Writer to the type, which is unpopulated.

File: mem/file.go [hunk 2]
```diff
@@ -218,6 +220,12 @@ func (f *File) WriteString(s string) (ret int, err error) {
 	return f.Write([]byte(s))
 }
 
+func (f *File) WriteTo(w io.Writer) (n int64, err error) {
```
Comment (davecheney) [inline mem/file.go]: This is not the signature for io.Writer, this is signature for io.WriterTo

**Comment by davecheney** [comment]
not lgtm sorry. The description doesn't match the change you've made in the code. I think you should raise an issue to discuss the problem you have before starting to code a solution.

Also, please include a test.

**Comment by chonthu** [comment]
Will close and discuss in issues first.

**Pull request closed without merging**